lab14/Task01-Seminar14.cpp

```diff
@@ -0,0 +1,32 @@
+﻿#include <iostream>
+const int MAX = 5;
+//Да се напише рекурсивна функция, която намира
+//сумата на елементите на масив от цели числа.
+void initialize(int* array, int size) {
+	for (int i = 0; i < size; i++)
+	{
+		std::cin >> array[i];
+	}
+}
+
+int sum(int* array, int index)
```

Comment:
It is better to pass the array's size as an argument to the function as well, instead of using the constant here - we would like our function to work properly for arrays of all sizes.